guest-js/ads/rewarded.ts

```diff
@@ -14,6 +15,16 @@ export class RewardedAd extends MobileAd<RewardedAdOptions> {
 	async show() {
 		return super.show();
 	}
+
+	async onRewarded(
+	  handler: (payload: any) => void
+  ): Promise<PluginListener> {
+    return await addPluginListener(
```

Comment:
Keep getting plugin not found error. [Discussion](https://github.com/tauri-apps/tauri/discussions/14520).